user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementValidationExecutor.java

```diff
@@ -1,9 +1,11 @@
 package life.qbic.datamanager.views.projects.project.measurements;
 
 import java.util.concurrent.CompletableFuture;
+import life.qbic.projectmanagement.application.measurement.MeasurementMetadata;
 import life.qbic.projectmanagement.application.measurement.validation.MeasurementValidationService;
 import life.qbic.projectmanagement.application.measurement.validation.ValidationResult;
 import life.qbic.projectmanagement.domain.model.project.ProjectId;
+import org.apache.poi.ss.formula.functions.T;
```

Comment:
This sound wrong...

Reply:
I think this import has to go. You wanted to use generics here and not some sort of Excel function?